wiki/component-development.md

```diff
@@ -87,10 +87,10 @@ If a component has subcomponents (`<OuiToolBar>` and `<OuiToolBarSearch>`), tigh
 Refer to the [SASS page][sass] of our documentation site for a guide to writing styles.
 
 [component-design]: component-design.md
-[docs]: https://elastic.github.io/eui/
+[docs]: https://oui.opensearch.org/
 [docs-yeoman]: creating-components-yeoman.md
 [docs-manual]: creating-components-manually.md
-[sass]: https://elastic.github.io/eui/#/guidelines/sass
+[sass]: https://oui.opensearch.org/1.0/#/guidelines/sass
```

Comment:
This one too?

Reply:
For this one, i tested `https://oui.opensearch.org/guidelines/sass`, it will show an error screen. But this URL `https://oui.opensearch.org/#/guidelines/sass`works, should I change it to this instead?

Reply:
Changed